### ISSUE TITLE
Helm buffer error!

### ISSUE THREAD

**Issue by achikin** [syl20bnr/spacemacs#10225]
#### Description :octocat:
Anything that uses helm buffer fails
#### Reproduction guide :beetle:
- Start Emacs
- Do anything that requires helm buffer e.g. SPC SPC
*Observed behaviour:* :eyes: :broken_heart:
Helm buffer does not show up, there is an error instead
```
helm-display-buffer: Wrong number of arguments: (lambda (buffer) "Display the Helm window respecting `dotspacemacs-helm-position'." (let ((display-buffer-alist (list spacemacs-helm-display-help-buffer-regexp spacemacs-helm-display-buffer-regexp))) (helm-default-display-buffer buffer))), 2
```
*Expected behaviour:* :heart: :smile:
Helm buffers function correctly
#### System Info :computer:
- OS: darwin
- Emacs: 25.3.1
- Spacemacs: 0.200.10
- Spacemacs branch: master (rev. 4bb4cb46)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(nginx org helm auto-completion better-defaults emacs-lisp git prodigy haskell markdown html go java sql
       (elm :variables elm-format-command "elm-format-0.18")
       javascript clojure evil-cleverparens themes-megapack)
```
- System configuration features: NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES

#### Backtrace :paw_prints:
```
helm-display-buffer: Wrong number of arguments: (lambda (buffer) "Display the Helm window respecting `dotspacemacs-helm-position'." (let ((display-buffer-alist (list spacemacs-helm-display-help-buffer-regexp spacemacs-helm-display-buffer-regexp))) (helm-default-display-buffer buffer))), 2
```

**Comment by achikin** [comment]
Also see #10219 and #10218

**Comment by achikin** [comment]
nevermind, it has been already fixed in #10196
